### PR TITLE
move to the latest rapidjson, and change the include path to rapidjson/

### PR DESCRIPTION
**Pull request by rny**
as stated https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin/issues/18

### CODE REVIEW

**Comment by minggo** [comment]
Sorry, where is new rapidjson?

**Comment by rny** [comment]
as stated https://code.google.com/p/rapidjson/

This repository has been moved to https://github.com/miloyip/rapidjson

**Comment by chengstory** [comment]
Thanks @rny.
@minggo  I think it may be merged!

**Comment by minggo** [comment]
@rny 
cocos2d-x doesn't have rapidjson libs.

**Comment by rny** [comment]
cocos2d-x does use rapidjson, it is header only, but yours(in the dependency) are an early version and it conflict with a later version (in the case that the project using the latest raidjson like us).  i will try to solve the conflicts tomorrow.

**Comment by rny** [comment]
@chengstory @minggo  I rebuild the patch to the latest version, and tested with the test build.

```
the headers is put into <rapidjson/*>, the same way the latest rapidjson library does.
```

File: cocos/3d/CCBundle3D.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ THE SOFTWARE.
 #include "renderer/CCGLProgram.h"
 #include "CCBundleReader.h"
 #include "base/CCData.h"
-#include "json/document.h"
+#include <rapidjson/document.h>
```
Comment (minggo) [inline cocos/3d/CCBundle3D.cpp:33]: We don't have rapidjson folder.

Comment (rny) [inline cocos/3d/CCBundle3D.cpp:33]: @chengstory @minggo you can pull it from https://github.com/miloyip/rapidjson

and removed the old /json

**Comment by zilongshanren** [comment]
@minggo  This PR could be closed now, since upgrade rapidjson is in our v3.7 milestone.

**Pull request closed without merging**